op_hello/src/hello.ts

```diff
@@ -0,0 +1,12 @@
+const OP_HELLO: number = Deno.ops.add("hello");
```

Comment:
So this is cool and I'd really like to use it that way, however due to quirks related to `shared_queue` I was unable to make this solution work - [see comments](https://github.com/denoland/deno/pull/3002#discussion_r326865393)

Reply:
Actually I'm gonna try to implement it once more, I'll get back to you

Reply:
Ah, ok - well let's not worry about the exact spelling of this - we can always adjust it in the future. How about this API: 
```js
export function hello() {
  Deno.core.send(Deno.ops["hello"]);
}
```
https://github.com/ry/deno/blob/op_hello/op_hello_js/hello.js

Reply:
@ry that will work (given that we "synchronize" ops on runtime initialization), however presented solution brings back @afinch7's concerns about performance hit caused by object lookup on each op call. Still we can start with this API and then figure out how to make it fast

Reply:
I'm quite sure we're not at the level of perf where a hash look up like that would effect our performance. I hope we are some day...

Reply:
Should I update #3002 to use this API?

Reply:
If you're in agreement, yea

Reply:
@ry PR updated

Reply:
In hindsight I think that hash lookup is fine, and I think my testing was flawed when I originally tested performance with a similar api.